interfaces/PA-App/src/app/personal-components/create-password/create-password.component.ts

```diff
@@ -126,7 +126,8 @@ export class CreatePasswordComponent extends PersonalComponent {
   }
 
   getNextSection() {
-    return PersonalComponents.createIdentity;
+    // return PersonalComponents.createIdentity;
+    return PersonalComponents.selectCountry;
```

Comment:
You can just remove the `createIdentity`-versoin
```suggestion
    return PersonalComponents.selectCountry;
```